jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/spi/RuntimeManagerLock.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Just a quesiton, why are not direclty using an custom implementation of java.util.concurrent.Lock that delegates into the real lock

Reply:
Or, just extends Lock and add methods of ReentrantLock that we are using

Reply:
because we need a way to ensure that other implementations can do other operations within....example of DebugRuntimeManagerLock which is the one taking data of the owner stacktrace that got the lock